<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QMainWindow" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="lblSearchState">
      <property name="text">
       <string>匹配数量：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblSearch">
      <property name="text">
       <string>搜索目录：</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="7">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="5">
     <widget class="QLineEdit" name="ledtSearchPath"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblPattern">
      <property name="text">
       <string>匹配格式：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="pbtnSearchBrowse">
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1" colspan="5">
     <widget class="QLineEdit" name="ledtBackupPath"/>
    </item>
    <item row="15" column="6">
     <widget class="QPushButton" name="pbtnBackupBrowse">
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="lblBackup">
      <property name="text">
       <string>备份目录：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="5">
     <widget class="QLineEdit" name="ledtPattern"/>
    </item>
    <item row="5" column="1" rowspan="5" colspan="5">
     <widget class="QListWidget" name="lstLogs"/>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="pbtnSearchCancel">
      <property name="text">
       <string>取消搜索</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="pbtnSearchStart">
      <property name="text">
       <string>开始搜索</string>
      </property>
     </widget>
    </item>
    <item row="17" column="6">
     <widget class="QPushButton" name="pbtnBackupCancel">
      <property name="text">
       <string>取消备份</string>
      </property>
     </widget>
    </item>
    <item row="17" column="5">
     <widget class="QPushButton" name="pbtnBackupStart">
      <property name="text">
       <string>开始备份</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="lblLogCount">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QPushButton" name="pbtnClearLogs">
      <property name="text">
       <string>清空列表</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="lblBackupState">
      <property name="text">
       <string>备份状态：</string>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QLabel" name="lblBackupInfo">
      <property name="text">
       <string>未开始</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
